server/main.cpp

```diff
@@ -64,7 +65,8 @@ int main(int argc, char* argv[])
 			}
 			// Parse the next argument for a port number.
 			char* end;
-			port = std::strtoul(argv[i], &end, 10);
+			commandPort = std::strtoul(argv[i], &end, 10);
+			eventPort = commandPort + 1;
```

Comment:
Do you want to have a new argument for the backchannel port instead of always taking the next one?

Reply:
I think this should be ok for now.
I explain better on my next comment.